### ISSUE TITLE
implement 'flags' keyword

### ISSUE THREAD

**Issue by stephen679** [vbatts/go-mtree#40]
```
[schung@localhost Desktop] $ sudo `which gomtree` -f mount.mtree -p /mnt
2016/07/25 19:43:01 Unknown keyword "flags" for file "."
```

**Comment by vbatts** [comment]
why closed?

**Comment by stephen679** [comment]
I realized there's a lot of other keywords not implemented, so I thought it'd make more sense to discuss which unimplemented keywords need attention, rather than just pointing out one.

**Comment by stephen679** [comment]
I could re-open and edit this issue accordingly if you want

**Comment by stephen679** [comment]
With regard to "important" keywords, and to align with upstream mtree, 'flags' and 'nlink' (tar archive only) are keywords that should be looked at next.

**Comment by vbatts** [comment]
sure thing. It's a good thing to implement these keywords. Though having them be included in the default set may not be needed. Especially ones like `nlink`, as technology like ostree use hardlinks heavily, so the `nlink` on disk will almost always be different on anything given host.

**Comment by vbatts** [comment]
`nlink` is already implemented

**Comment by stephen679** [comment]
yeah but for tar archives we would have to manually keep track of links

**Comment by vbatts** [comment]
oh i see. yeah, keeping track of hardlinks in tar archives is more work

**Comment by stephen679** [comment]
`nlink` keyword implemented for tar archives, although `nlink` for directory Entry's have not been implemented.

**Comment by vbatts** [comment]
I've just stubbed `flags`, but we can hold on these still.
